stuff/pipelines/rabbitmq_connection.py

```diff
@@ -7,12 +7,13 @@
 from typing import Iterator
 
 import argparse
+import logging
 import pika
 
 
 @contextlib.contextmanager
 def connect(
-        *, username: str, password: str, host: str
+        *, username: str, password: str, host: str, for_testing: bool
```

Comment:
Acá igual le puedes enviar un valor por defecto:
```suggestion
        *, username: str, password: str, host: str, for_testing: bool = False
```